### PR TITLE
feat: Admin actions columns to send verification emails

### LINKED ISSUE #5037: Admin: Add Action Column to send Password Reset and Activation link to users
Sometimes users contact us and say, they have not received the account verification or password reset link. 

Therefore on https://eventyay.com/admin/users/ please implement the following:
* add a column "Actions" with
  * a button to send a password reset email to the user
  * Only for Unverified Users show a button to resend the "Verification Email"

Screenshot from 2020-09-09 22-31-12

### PR DESCRIPTION
**Pull request by g4rry420**
Fixes #5037

#### Short description of what this resolves:

Added the column Action in the /admin/users/all panel.

#### Changes proposed in this pull request:

- Admin can send email to unverified users by clicking a button to verify it.
-Admin can also send reset email to users by clicking a button.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by g4rry420** [comment]
@iamareebjamal Sir, I am new to the ember environment, please let me know if I have made any mistakes.

**Comment by maze-runnar** [comment]
share working video.

**Comment by g4rry420** [comment]
https://user-images.githubusercontent.com/61521805/113236306-9e9dc980-9272-11eb-92f7-c10b9c3a7f90.mp4


Local server is not sending email to the email address. I am not sure if this is expected or not. Please let me know.

**Review by MeghalBisht — APPROVED** [review]
Works!

**Review by MeghalBisht — APPROVED** [review]

File: app/templates/components/ui-table/cell/admin/users/cell-user-actions.hbs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <div class="item">
+      <button type="button" class="w-full ui button" {{action @props.actions.sendVerificationMail @extraRecords.email}}>
+        {{t "Resend Verification Email"}}
+      </button>
+    </div>
+  {{/unless}}
+  <div class="item">
+    <button type="button" class="w-full ui button" {{action @props.actions.resetPasswordMail @extraRecords.email}}>
+      {{t "Password Reset Email"}}
+    </button>
```
Comment (maze-runnar) [inline app/templates/components/ui-table/cell/admin/users/cell-user-actions.hbs:12]: @mariobehling the text `Resend Verification Email` is fine here, or it need to be replaced with mail icon?

**Review by maze-runnar — APPROVED** [review]

**Review by ranjitkshah — APPROVED** [review]

**Review by sachinchauhan2889 — APPROVED** [review]

**Pull request merged**